tests/rust_libs/tests/01-run.py

```diff
@@ -48,11 +39,25 @@ def check_cmd(child, cmd, expected):
         else:
             child.expect_exact(line)
 
+def check_cmd_list(child):
+    child.expect(PROMPT)
+    child.sendline('cmds_json')
+    child.expect(r"(\{[^\n\r]*\})\r\n")
```

Comment:
The native output is `\n`, but the python test automation framework converts this internally before matching. It would probably been better to figure out a way to configure the framework to not change line endings, but that would require updating a lot of test scripts; so rather out of scope here IMO.

Reply:
Yeah, definitely out of scope, I was just wondering :)